tools/importer/rules/zPattern.js

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Please indent the code properly and run eslint as well

Reply:
done.